registrar.go

```diff
@@ -119,9 +119,11 @@ func (r Registry) FilterForCompatible(ctx context.Context) Drivers {
 
 	var result Drivers
 	for i := 0; i < len(state); i++ {
-		if state[i] != nil {
-			result = append(result, state[i])
+		if state[i] == nil {
+			continue
 		}
+
+		result = append(result, state[i])
```

Comment:
curious how these two are different?

Reply:
yep - its the same, this was just a trivial readability fix